deployer/src/deployment/state_change_layer.rs

```diff
@@ -372,14 +222,14 @@ mod tests {
         state: State,
     }
 
-    impl From<Log> for StateLog {
-        fn from(log: Log) -> Self {
-            Self {
-                id: log.id,
-                state: log.state,
-            }
-        }
-    }
+    // impl From<LogItem> for StateLog {
+    //     fn from(log: LogItem) -> Self {
+    //         Self {
+    //             id: log.id,
+    //             state: log.state,
+    //         }
+    //     }
+    // }
```

Comment:
```suggestion
```

Reply:
Might need it back if we're updating those unit tests